Content.Client/Preferences/UI/HumanoidProfileEditor.xaml.cs

```diff
@@ -498,31 +498,40 @@ private void RebuildSpriteView()
 
             var sprite = _entMan.GetComponent<SpriteComponent>(_previewDummy);
```

Comment:
Looking at the line above this, this will currently just keep on spawning new entities anytime the dummy needs to be updated, and when the window gets disposed, it only deletes the most recently created entity. So moving the slider around will spawn hundreds of entities? Actually maybe thousands given that each mob has limbs, organs, etc.

Really this should probably be changed so that RebuildSpriteView() is only ever called if the species selection changes, and If it does have to spawn a new entity, it should delete the existing one (if there is one).

So yeah, maybe cache the result of `Profile?.Species ?? SpeciesManager.DefaultSpecies` and check if it has changed (or was previously null)?